Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/ReflectionAdapter.cpp

```diff
@@ -643,140 +643,88 @@ namespace AZ::DocumentPropertyEditor
-            const auto applyAttributeChangeExcludingSiblings =
-                [&](Dom::Value& parentNode, Dom::Path& parentPath, AZStd::function<void(const Dom::Value&, Dom::Path&)> procedure)
-            {
-                procedure(parentNode, parentPath);
-                if (auto disabledAttributeIter = parentNode.FindMember(disabledAttributeName);
-                    disabledAttributeIter != parentNode.MemberEnd() && !disabledAttributeIter->second.GetBool())
-                {
-                    queueDescendantsForSearch(parentNode, parentPath);
-                }
-            };
-
-            // Call this for cases where the attribute change should propagate from the origin node's parent, including all sibling trees
-            const auto propagateAttributeChangeToSiblings =
-                [&](Dom::Value& parentNode, Dom::Path& parentPath, AZStd::function<void(const Dom::Value&, Dom::Path&)> procedure)
+            const auto propagateAttributeChangeToRow = [&](
+                const Dom::Value& parentNode,
+                const Dom::Path& parentPath,
+                AZStd::function<void(const Dom::Value&, const Dom::Path&)> procedure)
             {
-                // Remove the disabled attribute from the origin node and its property editor siblings
                 int index = 0;
                 for (auto child = parentNode.ArrayBegin(); child != parentNode.ArrayEnd(); ++child)
                 {
                     if (child->IsNode())
                     {
-                        Dom::Path childPath = parentPath / index;
-
-                        if (child->GetNodeName() == GetNodeName<Nodes::PropertyEditor>())
+                        auto childPath = parentPath / index;
+                        if (child->GetNodeName() != GetNodeName<Nodes::Row>())
                         {
                             procedure(*child, childPath);
-
-                            if (auto disabledAttributeIter = child->FindMember(disabledAttributeName);
-                                disabledAttributeIter != child->MemberEnd() && !disabledAttributeIter->second.GetBool())
-                            {
-                                queueDescendantsForSearch(*child, childPath);
-                            }
-                        }
-                        else if (child->GetNodeName() == GetNodeName<Nodes::Row>())
-                        {
-                            queueDescendantsForSearch(*child, childPath);
                         }
+                        queueDescendantsForSearch(*child, childPath);
                     }
                     ++index;
                 }
             };
 
-            // Applies the attribute change to any descendants in unvisitedChildren until its done
-            const auto propagateAttributeChangeToDescendants = [&](AZStd::function<void(Dom::Path&)> procedure)
+            // This lambda applies the attribute change to any descendants in unvisitedChildren until its done
```

Comment:
Typo here: unvisitedChildren -> unvisitedDescendants
Should probably just call it something less variable name-y; will pick that up in another PR if AR succeeds and there aren't any other comment.s